packages/nitro-protocol/gas-benchmarks/vanillaSetup.ts

```diff
@@ -49,3 +60,25 @@ beforeEach(async () => {
 afterAll(async () => {
   await ganacheServer.close();
 });
+
+expect.extend({
+  async toConsumeGas(
+    received: any, // TransactionResponse
```

Comment:
sadly [this type](https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse) does not seem to be exported by ethers.